discovery.go

```diff
@@ -174,6 +174,11 @@ func (r *Router) DiscoveryAllBuckets(ctx context.Context) error {
 					knownBucket.Add(1)
 				}
 
+				// if TotalBucketCount < BUCKET_CHUNK_SIZE then nextFrom equal zero
+				if rawReq.From == *nextFrom {
```

Comment:
I tried to get into this condition, but unfortunately I didn’t succeed, can you please refer to the line of code from the original lua vshard storage/router where it can implement this behavior or describe the situation

Reply:
this code fixes a bag, which presents in next case:
if tarantool cluster has total bucket's count < 1000
Bag performs when go-router discovers all buckets with batching method, in first iteration router send command "discovers_bucket" to cluster, then tarantool responses with bucekt and nextFrom, in this case nextFrom equals zero because tarantool discover all buckets and batching is not needed after that
if total bucket's count > 1000 then tarantool send nextFrom > 0 to perform client select next bucekts